tests/transformers/test_scale.py

```diff
@@ -0,0 +1,134 @@
+from pandas import DataFrame
+from sklearn.preprocessing import MinMaxScaler as SklearnMinMaxScaler
+from sklearn.preprocessing import Normalizer as SklearnNormalizer
+from sklearn.preprocessing import StandardScaler as SklearnStandardScaler
+
+from pyreal.transformers.scale import MinMaxScaler, Normalizer, StandardScaler
+
+# MinMaxScaler tests
+data = [[2, 1, 3, 10], [4, 3, 4, 0], [6, 7, 2, 2]]
```

Comment:
Generally we want to avoid global variables like this in unit tests - either use the data from the conftest fixture you added (`test_fit_minmax(minmax_scale_data)`), or if it needs to be different for each unit test, set it at the beginning of the test itself